tests/runner.py

```diff
@@ -68,6 +68,21 @@ def path_from_root(*pathelems):
 
   python tests/runner.py random50
 
+An individual test can be skipped by passing the "skip:" prefix. E.g.
```

Comment:
Hmm, I see the html docs are a bit more prose, so added a bit more verbose paragraphs there.